pkg/cli/admin/release/extract_tools.go

```diff
@@ -457,10 +462,11 @@ func (o *ExtractOptions) extractCommand(command string) error {
 			NewArch: true,
 			Mapping: extract.Mapping{Image: "installer-artifacts", From: "usr/share/openshift/linux_arm64/openshift-install"},
 
-			Readme:               readmeInstallUnix,
-			InjectReleaseImage:   true,
-			InjectReleaseVersion: true,
-			ArchiveFormat:        "openshift-install-linux-arm64-%s.tar.gz",
+			Readme:                    readmeInstallUnix,
+			InjectReleaseImage:        true,
+			InjectReleaseVersion:      true,
+			InjectReleaseArchitecture: true,
+			ArchiveFormat:             "openshift-install-linux-arm64-%s.tar.gz",
```

Comment:
Why https://github.com/openshift/oc/blob/4644b093bb491e843eaa422e880d87a5c2235de5/pkg/cli/admin/release/extract_tools.go#L485-L495 is not in this list?

Reply:
This is an oversight on my part, thanks for catching. will include in my next commit.